components/next_steps_view/next_steps_view.scss

```
@@ -0,0 +1,88 @@
+#app-content.NextStepsView {
+    background-image: url(../../images/onboarding-bg.svg);
+    background-repeat: no-repeat;
+    background-position-y: bottom;
+    background-position-x: -305px;
+    background-size: calc(100% + 305px);
+}
+
+.NextStepsView__header {
+    padding: 32px 40px;
+    display: flex;
+}
+
+.NextStepsView__header-headerText {
+
+}
+
+.NextStepsView__header-headerTopText {
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 40px;
+    letter-spacing: -0.02em;
+}
+
+.NextStepsView__header-headerBottomText {
+    font-size: 16px;
+    line-height: 24px;
+    margin-top: 8px;
+}
+
+.NextStepsView__header-logo {
+    margin-left: auto;
+    align-self: center;
+}
+
+.NextStepsView__body {
+    display: flex;
+    height: 100%;
+}
+
+.NextStepsView__body-main {
+    flex: 3 0 75%;
+}
+
+.NextStepsView__body-graphic {
+    background-image: url(../../images/getting-started.svg);
+    background-repeat: no-repeat;
+    background-size: 120%;
+    background-position-y: 33%;
+    flex: 1 0 25%;
+    z-index: 2;
+    min-width: 192px;
+}
+
+@media screen and (max-width: 768px) {
+    #app-content.NextStepsView {
+        background-size: auto;
+        background-position-y: top;
```

From my comment above:

```suggestion
        background-position-y: bottom;
```